src/parser/shared/modules/items/bfa/FirstMatesSpyglass.js

```diff
@@ -74,15 +79,22 @@ class FirstMatesSpyglass extends Analyzer {
       return this.selectedCombatant.getBuffUptime(SPELLS.SPYGLASS_SIGHT.id) / this.owner.fightDuration;
   }
 
-  item() {
-    return {
-      item: ITEMS.FIRST_MATES_SPYGLASS,
-      result: (
-        <TooltipElement content={`You critically hit ${formatPercentage(this.timesCrit / this.timesHit)}% of the time with this buff up`}>
-          Used {this.casts} times / {formatPercentage(this.totalBuffUptime)}% uptime
-        </TooltipElement>
-      ),
-    };
+  get critPercentage() {
+    return this.timesCrit / this.timesHit;
+  }
+
+  statistic() {
+    return (
+      <ItemStatistic
+        size="flexible"
+        tooltip={`Used ${this.casts} times`}
+      >
+        <BoringItemValueText item={ITEMS.FIRST_MATES_SPYGLASS}>
+          <UptimeIcon /> {formatPercentage(this.totalBuffUptime)}% <small>uptime</small><br />
+          <CritIcon /> {formatPercentage(this.critPercentage,0)}% <small>critical hits during buff</small>
```

Comment:
Since this isn't info the player can directly use it should be a part of the tooltip. An interesting stat might be the average crit gained.

Reply:
The only reason i swapped it from the tooltip into the box was because i thought it was more interesting than the number of times it was used. but i can switch it back.